src/github-repo.ts

```diff
@@ -244,7 +244,17 @@ export class GithubRepo {
     }
     // Doing in sequence to not overload GitHub with requests
     for (const asset of assets) {
-      await this._uploadAsset(releaseDetails, asset);
+      try {
+        await this._uploadAsset(releaseDetails, asset);
+      } catch (err) {
+        if ((err as Error)?.message?.includes('ECONNRESET')) {
```

Comment:
There's probably a better way to check for this error, maybe an error code?

Reply:
If I understand the logs in evergreen correctly, this error object has a `.status` property with a value of `500`, it might make sense to check for that (or 5xx errors in general)?